docs/tutorials/12_deuteron_binding_energy.ipynb

```diff
@@ -37,15 +37,15 @@
     "import numpy as np\n",
     "import matplotlib.pyplot as plt\n",
     "from IPython.display import display, clear_output\n",
-    "from qiskit_aer import Aer\n",
-    "from qiskit.algorithms import VQE\n",
-    "from qiskit.algorithms.aux_ops_evaluator import eval_observables\n",
+    "from qiskit.primitives import Estimator\n",
+    "from qiskit.algorithms.minimum_eigensolvers import VQE\n",
+    "from qiskit.algorithms.observables_evaluator import estimate_observables\n",
     "from qiskit.algorithms.optimizers import COBYLA, SLSQP\n",
     "from qiskit.circuit import QuantumCircuit, Parameter\n",
     "from qiskit.circuit.library import TwoLocal\n",
-    "from qiskit.opflow import I, X, Y, Z, PauliSumOp\n",
-    "from qiskit.opflow.expectations import PauliExpectation\n",
-    "from qiskit.utils import QuantumInstance, algorithm_globals\n",
+    "from qiskit.opflow import PauliSumOp\n",
```

Comment:
Are we able to also already get rid of `PauliSumOp` here?

Reply:
Yes, I remove it from the tutorial. The convert method still returns PauliSumOp but I extract the SparsePauliOp from it.

Reply:
Yeh I suppose this is fine for now. Once we resolve #879 we can also change this piece of code to use the `JordanWignerMapper` directly rather than wrap it with a `QubitConverter` :+1:
This should happen prior to v0.6 anyways